resdk/analysis/__init__.py

```diff
@@ -1 +1,2 @@
 """Collection of utility functions for running analysis."""
+from . import _register
```

Comment:
Why is this needed?

Reply:
Originally, `resdk/__init__.py` patched the resources using:
```from .analysis import _register```

I wanted to similarly patch from `.data_uploads`, but this confused the namespace of `_register`. So, I instead had to patch using:
```
from . import analysis
from . import data_upload
```
and then include the `_register` imports in the sub-modules.

Reply:
This complicates things, you should probably do

    from .analysis import _register as analysis_register
    from .data_upload import _register as upload_register

and even importing just `_register`-s should probably work, only pylint will complain about.